dashboard/management/commands/createuser.py

```diff
@@ -1,5 +1,6 @@
 from django.core.management.base import BaseCommand
 from django.contrib.auth.models import User
+from django.conf import settings
 import random, string
 
 RANDOM_PASSWORD_DEFAULT_LENGTH = 8
```

Comment:
Delete this line.  With the new import, the old `RANDOM_PASSWORD_DEFAULT_LENGTH` is no longer used.